docs/docker_host_ssh.md

```diff
@@ -0,0 +1,116 @@
+Reuse docker host SSH daemon
+============================
+
+It is possible to use the SSH daemon that runs on the docker host instead of using a separate SSH port for GitLab.
+
+## Setup
+
+Create the system `git` user if necessary (make sure it has a working shell):
+
+```bash
+useradd --system git --home-dir /srv/docker/gitlab/data
+```
+
+The home directory must point to GitLab data directory, as it takes `.ssh/authorized_keys` from there.
+
+Now create `gitlab-shell` home directory at the same location as in the container:
+
+```bash
+mkdir -p /home/git/gitlab-shell/bin
```

Comment:
Right, I finally realized that this guide was not for the official Docker images :) Sorry for the noise.

I noticed it when I realized the `USERMAP_UID` and `USERMAP_GID` variables were not being respected.

I'll have to figure out if I can do something similar with the offical images, because this "hack" to be able to use port 22 for git is really neat (thanks!).